mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.java

```diff
@@ -654,6 +668,96 @@ public void onClick(View view) {
         }
     }
 
+    public static class HabpanelFragment extends Fragment implements
+            ConnectionFactory.UpdateListener {
+        Connection mConnection;
+
+        public static HabpanelFragment newInstance() {
+            return new HabpanelFragment();
+        }
+
+        @Override
+        public View onCreateView(@NonNull LayoutInflater inflater,
+                ViewGroup container, Bundle savedInstanceState) {
+            View view = inflater.inflate(R.layout.fragment_habpanel, container, false);
+
+            TextView retryButton = view.findViewById(R.id.retry_button);
+            retryButton.setOnClickListener(v -> loadHabpanel(view));
+            loadHabpanel(view);
```

Comment:
Should still move those 3 lines to onViewCreated(), so `updateViewVisibility()` from `loadHabpanel()` works. Can omit the `view` parameter from `loadHabpanel` in that case as well.

Reply:
Done :)